src/main/java/uk/ac/ox/oxfish/fisher/equipment/fads/FadManager.java

```diff
@@ -79,7 +81,6 @@ public FadManager(
             new HashSet<>(),
```

Comment:
Should we worry about this being a Hash rather than a LinkedHash? Don't these get iterated over?

Speaking of: is there a test to make sure that, say, a 2 year long run with TunaScenario when ran twice with the same seed outputs the same results?

Reply:
Good point. I'll change it. Also added a note about testing to https://github.com/poseidon-fisheries/tuna/issues/29.